Job/DoctrineJobRepository.php

```diff
@@ -65,9 +71,20 @@ public function getJobManager()
     /**
      * {@inheritdoc}
      */
+    public function getJobExecution(JobInstance $jobInstance)
```

Comment:
And doing that in the createJobExecution will allow not to add something else to the interface.

Or do you have a usage of the method outside ? As you put it public ?